src/pages/index.js

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { Link } from "gatsby"
+import styles from "./styles/index.module.css"
+import { graphql } from "gatsby"
+
+import Layout from "../components/layout"
+
+export default function Home(props) {
```

Comment:
For components that don't have any logic and just return a value, sometimes it's nicer/shorter to write them with arrow functions (which don't require a `return` to return their value):

```js
export default props => (
  <Layout>
    ...
  </Layout>
)
```

@solomonhawk I might be forgetting — is there a downside to exporting a default function that has no name like I'm doing in my example? I do this pretty frequently and haven't run into issues, but I know some devs prefer to name them + then export them.